vignettes/HOWTO-BCV.rmd

```diff
@@ -0,0 +1,150 @@
+---
+title: "HOWTO generate biocViews HTML"
+author: "S. Falcon and V.J. Carey"
+date: "`r format(Sys.time(), '%B %d, %Y')`"
+vignette: >
+  %\VignetteIndexEntry{Quick-start Guide}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+output:
+  BiocStyle::html_document:
+    number_sections: no
+    toc: yes
+    toc_depth: 2
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+
+```{r include=FALSE}
+
+if(!require(biocViews)){
+    install.packages("biocViews")
```

Comment:
As @LiNk-NY mentioned in https://github.com/Bioconductor/biocViews/pull/4#issuecomment-1150207341, we want to keep this close to the original as possible. We also don't typically `install` in vignettes. Assume that it's already installed and that we just need to use the `library` function.